src/ucp/core/ucp_worker.c

```diff
@@ -2906,8 +2909,8 @@ ucp_worker_keepalive_next_ep(ucp_worker_h worker)
 
     worker->keepalive.iter = worker->keepalive.iter->next;
     if (worker->keepalive.iter == &worker->all_eps) {
-        /* if next list item points to all_eps then step one more time */
-        worker->keepalive.iter = worker->keepalive.iter->next;
+        worker->keepalive.lane_map = 0;
```

Comment:
maybe instead of 2 functions (ucp_worker_keepalive_current_ep, ucp_worker_keepalive_next_ep) - we have one function 'ucp_worker_keepalive_iter_next_ep()' which : 1. returns current ep , and 2. moves iterator to next ep

Reply:
done